### PR TITLE
Delivery readme fixes

### PR DESCRIPTION
**Pull request by pericsson**
Updated:
* `DELIVERY.README.MethylSeq.txt`
* `DELIVERY.README.RAW_DATA.txt`
* `DELIVERY.README.RNASeq.txt`
* `DELIVERY.README.Sarek.txt`
  * Moved the samples out of 00-reports to match the PR for the [sarek delivery yaml, bimonthly branch](https://github.com/NationalGenomicsInfrastructure/irma-provision/pull/412)

### CODE REVIEW

**Review by jun-wan — COMMENTED** [review]
Looked good to me.

**Comment by pericsson** [comment]
The sample folder was also now moved out of `01-RNA-Results` for the RNA-seq delivery readme.

File: delivery_readmes/DELIVERY.README.RNASeq.txt [hunk 1]
```diff
@@ -15,17 +15,17 @@ Use the md5 checksums to verify the integrity of the files after transfer.
...
-   │   ├── Sample2.md5
+   ├── Sample1/
+   ├── Sample1.lst
+   ├── Sample1.md5
+   ├── Sample2/
+   ├── Sample2.lst
+   ├── Sample2.md5
    ...
    ...
    │   ├── SampleN/
```
Comment (jun-wan) [inline delivery_readmes/DELIVERY.README.RNASeq.txt:26]: "│   ├── SampleN/" needs to be moved out of 01-RNA-Results/ as well.

Comment (pericsson) [reply delivery_readmes/DELIVERY.README.RNASeq.txt]: Thanks, I'll change this

**Review by aanil — APPROVED** [review]
👍

**Comment by pericsson** [comment]
Also modified `DELIVERY.README.MethylSeq.txt` and `DELIVERY.README.RAW_DATA.txt`.

File: delivery_readmes/DELIVERY.README.MethylSeq.txt [hunk 2]
```diff
@@ -39,14 +39,14 @@ The 00-Report folder contains reports with sequencing information about the runs
...
-   └── manifestFiles/
-       └── <NGI_ID>.02-FASTQ.<flowcell>.<NGI_ID>_<flowcell-mode>_<lane-number>_manifest.txt
+       ├── ProjectName_lanes_info.txt
+       ├── ProjectName_library_info.txt
+       ├── ProjectName_multiqc_report.html
+       ├── ProjectName_project_summary.html
+       ├── ProjectName_project_summary.md
+       ├── ProjectName_sample_info.txt
+       └── manifestFiles/
+            └── <ProjectID>.02-FASTQ.<flowcell>.<ProjectID>_<BCL-CONVERSION-ID>_<lane-number>_manifest.txt
```
Comment (jun-wan) [inline delivery_readmes/DELIVERY.README.MethylSeq.txt:49]: Sorry, my bad,  "ProjectID" should be "SampleID" here actually.

File: delivery_readmes/DELIVERY.README.MethylSeq.txt [hunk 3]
```diff
@@ -64,10 +64,10 @@ A report summarising the information about the run, samples, and library.
 --ProjectName_project_summary.md
 The .md file of the project_summary.html file used to generate the report.
 
- --ProjectName_sample_info.txt
+--ProjectName_sample_info.txt
 A file with information about the samples such as names, million reads, and Q30 values.
 
---<NGI-ID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
+--<ProjectID>.02-FASTQ.<flowcell>.<ProjectID>_<BCL-CONVERSION-ID>_<lane-number>_manifest.txt
```
Comment (jun-wan) [inline delivery_readmes/DELIVERY.README.MethylSeq.txt:70]: Sorry, same as above, "ProjectID" -> "SampleID" here.

File: delivery_readmes/DELIVERY.README.MethylSeq.txt [hunk 4]
```diff
@@ -87,7 +87,7 @@ Contains a subfolder for each flowcell, which in turn contains the FastQ files f
 
 ── 02-FASTQ/
    └── <flowcell>/
-       └── <NGI-ID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
+       └── <ProjectID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
```
Comment (jun-wan) [inline delivery_readmes/DELIVERY.README.MethylSeq.txt:90]: Sorry, "ProjectID" -> "SampleID" here as well.

File: delivery_readmes/DELIVERY.README.RAW_DATA.txt [hunk 5]
```diff
@@ -39,35 +57,35 @@ The 00-Report folder contains reports with sequencing information about the runs
...
-   └── manifestFiles/
-       └── <NGI_ID>.02-FASTQ.<flowcell>.<NGI_ID>_<flowcell-mode>_<lane-number>_manifest.txta 
+       ├── ProjectName_lanes_info.txt
+       ├── ProjectName_library_info.txt
+       ├── ProjectName_multiqc_report.html
+       ├── ProjectName_project_summary.html
+       ├── ProjectName_project_summary.md
+       ├── ProjectName_sample_info.txt
+       └── manifestFiles/
+            └── <ProjectID>.02-FASTQ.<flowcell>.<ProjectID>_<BCL-CONVERSION-ID>_<lane-number>_manifest.txt
```
Comment (jun-wan) [inline delivery_readmes/DELIVERY.README.RAW_DATA.txt:67]: Sorry, "ProjectID" -> "SampleID".

File: delivery_readmes/DELIVERY.README.RAW_DATA.txt [hunk 6]
```diff
@@ -39,35 +57,35 @@ The 00-Report folder contains reports with sequencing information about the runs
...
 
 --ProjectName_project_summary.md
 The .md file of the project_summary.html file used to generate the report.
 
- --ProjectName_sample_info.txt
+--ProjectName_sample_info.txt
 A file with information about the samples such as names, million reads, and Q30 values.
 
---<NGI-ID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
+--<ProjectID>.02-FASTQ.<flowcell>.<ProjectID>_<BCL-CONVERSION-ID>_<lane-number>_manifest.txt
```
Comment (jun-wan) [inline delivery_readmes/DELIVERY.README.RAW_DATA.txt:88]: Sorry, "ProjectID" -> "SampleID".

File: delivery_readmes/DELIVERY.README.RAW_DATA.txt [hunk 7]
```diff
@@ -87,4 +105,4 @@ Contains a subfolder for each flowcell, which in turn contains the FastQ files f
 
 ── 02-FASTQ/
    └── <flowcell>/
-       └── <NGI-ID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
+        └── <ProjectID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
```
Comment (jun-wan) [inline delivery_readmes/DELIVERY.README.RAW_DATA.txt:108]: "ProjectID" -> "SampleID".

File: delivery_readmes/DELIVERY.README.Sarek.txt [hunk 8]
```diff
@@ -55,14 +55,14 @@ ProjectID -> 00-Reports
...
-    └── manifestFiles/
-        └── <ProjectID>.02-FASTQ.<flowcell>.<ProjectID>_<BCL-CONVERSION-ID>_<lane-number>_manifest.txt
+   ├── ProjectName_project_summary.html
+   ├── ProjectName_sample_info.txt
+   ├── ProjectName_lane_info.txt
+   ├── ProjectName_library_info.txt
+   ├── ProjectName_multiqc_report.html
+   ├── ProjectName_SarekGermline_multiqc_report.html
+   └── manifestFiles/
+         └── <ProjectID>.02-FASTQ.<flowcell>.<ProjectID>_<BCL-CONVERSION-ID>_<lane-number>_manifest.txt
```
Comment (jun-wan) [inline delivery_readmes/DELIVERY.README.Sarek.txt:65]: Sorry, "ProjectID" -> "SampleID"

File: delivery_readmes/DELIVERY.README.Sarek.txt [hunk 9]
```diff
@@ -99,4 +99,4 @@ Contains a subfolder for each flowcell, which in turn contains the FASTQ files.
 
 ├──02-FASTQ/
    └── <Flowcell>/
-       └── <ProjectID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
+         └── <ProjectID>_<BCL-conversion-ID>_<lane-number>_<read-number>_<volume>.fastq.gz
```
Comment (jun-wan) [inline delivery_readmes/DELIVERY.README.Sarek.txt:102]: Sorry, "ProjectID" -> "SampleID"

File: delivery_readmes/DELIVERY.README.MethylSeq.txt (hunk 2 above)
Comment (pericsson) [reply delivery_readmes/DELIVERY.README.MethylSeq.txt:49]: No problem, I didn't realise that. I'll fix it!

**Review by jun-wan — COMMENTED** [review]
In DELIVERY.README.RNASeq.txt line 65, 76 and 101, <ProjectID> should be <SampleID>, my bad, sorry!

**Review by jun-wan — COMMENTED** [review]
Great work! Everything looked fine for me.

**Review by parlundin — APPROVED** [review]
Looks fine as far as I can see.

**Pull request merged**